README.md

```diff
@@ -90,6 +90,54 @@ result = client.match_spectra_from_path(
 #### Notebooks
 You can find a [tutorial](https://github.com/omigami/omigami/blob/master/notebooks/ms2deepscore/tutorial.ipynb) notebook in the `/notebooks/` folder.
 
+#### Plotting
+Plotting graphs works the same way for both Spec2Vec and MA2DeepScore.
+
+
+The following example will plot the structures of the molecules
+```python
+from omigami import MoleculePlotter
+plotter = MoleculePlotter()
+plotter.plot_molecule_structure_grid(best_matches, 
+                                     draw_indices=True, 
+                                     molecule_image_size=[600, 600], 
+                                     substructure_highlight="C(=O)")
+```
+
+
+#### Plotting
+Plotting works the same way for both Spec2Vec and MS2DeepScore.
+
+
+The following example will plot the structures of the molecules of the best matches.
+```python
+from omigami import MoleculePlotter
+plotter = MoleculePlotter()
+plotter.plot_molecule_structure_grid(best_matches, 
+                                     draw_indices=True, 
+                                     molecule_image_size=[600, 600], 
+                                     substructure_highlight="C(=O)")
+```
+<img src="docs/images/molecule_plot.png " width="500">
+
+
+Want to plot the results of the [Classyfire](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-016-0174-y) model. The following code will handle that for you.
```

Comment:
I still don't like this sentence. It seems you are trying to sell something and it is not the same "tone" as the rest of the documentation

Reply:
Maybe just say "The following code allows us to plot the results of the ..."